nav2_dynamic_msgs/msg/Obstacle.msg

```diff
@@ -3,3 +3,5 @@ float32 score # detection confidence
 geometry_msgs/Point position # center
 geometry_msgs/Vector3 velocity
 geometry_msgs/Vector3 size
+geometry_msgs/Vector3 position_covariance
```

Comment:
I think these are usually represented by a dense matrix, no?

Reply:
The radar that I'm using presently only gives me RMS in only the the 3 linear degrees of freedom. However it wouldn't be difficult to make this adaptable for radars that have cross terms in the covariance matrix. Should I use a `float64[9]` for this?

Reply:
Yeah for both covars so that its in line with the same in other sensor message types

Reply:
fixed in 5af5d49300af9cd023721984c2fae560ae49daee